attributes/network/9_account_features/cddl/1_account_multisig.cddl

```diff
@@ -19,10 +19,12 @@ account-multisig-transaction-submit = {
     0 => [9, 1, 0],
     1 => non-anonymous-identity,        ; Submitter
     2 => non-anonymous-identity,        ; Account source
-    3 => text,                          ; Description of the transaction
+    ? 3 => bstr,                        ; Description of the transaction
```

Comment:
Why not a `tstr`?

Reply:
Let's have both a memo line (`tstr`) and a `metadata` line (`bstr`).